gcp_variant_transforms/libs/annotation/vep/vep_runner.py

```diff
@@ -0,0 +1,464 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from typing import Dict, List, Any  # pylint: disable=unused-import
+
+import argparse  # pylint: disable=unused-import
+import logging
+import time
+
+from apache_beam.io import filesystems
+from apache_beam.io import filesystem  # pylint: disable=unused-import
+from apache_beam.options import pipeline_options
+from googleapiclient import discovery
+from oauth2client import client
+
+
+# Minimum size of virtual machine disks to account for cache files.
+# NOTE(bashir2): The proper way of setting this is to measure the input cache
+# file size and use some heuristics to calculate the size of the decompressed
+# version. The current value is more than double what is needed for 'human'.
+_MINIMUM_DISK_SIZE_GB = 200
+
+# The name of the file placed at the root of output_dir that includes
+# information on how the pipelines were run, input files, etc.
+_GLOBAL_LOG_FILE = 'VEP_run_info.txt'
```

Comment:
Done.
But please note that this file is currently empty and solely used as a lock to prevent overwrites (there is a TODO below to put some the run information in it). The actual log files are `output_VM*` (check `_get_output_log_path` below). I added `.log` extension there as well.